src/main/java/com/timelogr/enterprise/TimeLogrServices.java

```diff
@@ -1,27 +1,25 @@
 package com.timelogr.enterprise;
 
 import org.springframework.stereotype.Service;
-import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.GetMapping;
 
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * TESTING DUMMY DATA
+ */
```

Comment:
Added spacing and converted the comment in this file to a JavaDoc: https://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html